atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceTableCreationIntegrationTest.java

```diff
@@ -166,6 +173,29 @@ public void testGetMetadataCaseInsensitive() {
         kvs.dropTable(caseSensitiveTable);
     }
 
+    @Test
+    public void testCreateTransactions2TableProducesCorrectMetadata() throws TException {
```

Comment:
it's somewhat a weird test imo, what happens when you need to change it? you'll just edit the test. Happy to keep here, but unclear what value it has.

Reply:
This was meant to test the changes to `CassandraTableCreator` (to set the new options) which itself doesn't have tests right now. So if I say forgot to write line 91 (`SchemaBuilder....maxIndexInterval(...)`) for the table-creator or someone removes it later on, this test should catch that. We could conceivably check that the CQL query there is well formed, though I think an integration test would remain useful.

Reply:
That's fair.